speedy/extracting_speedy.jl

```diff
@@ -0,0 +1,126 @@
+using FortranFiles
```

Comment:
Tuomas pointed out this file may not actually be used anywhere.

Reply:
Yes this is correct, I can remove this. I was using it to extract the speedy outputs and then saving them to a .h5 file